udata/core/dataset/views.py

```diff
@@ -103,6 +104,19 @@ def get_context(self):
         return context
 
 
+@blueprint.route('/r/<uuid:id>', endpoint='resource')
```

Comment:
Alright, I misinterpreted where that route was issued, I thought it was at the root like directly `data.gouv.fr/r/cc3cb0d0-87ed-4eb6-aced-470c083741cd` I wonder if it is required to nest it. What about a more general shortcut to access orgs, datasets and resources (probably out of the scope of that PR)